saos-webapp/src/main/webapp/WEB-INF/static/javascript/analysis.js

```diff
@@ -1,80 +1,217 @@
+    function isAnalysisFormEmpty() {
+        var empty = true;
+        $("#analysisForm :text").each(function() {
+            if($(this).val() !== "") {
+                empty = false;
+                return;
+            }
         });
+        
+        return empty;
+    }
+    
+    /**
+     * Prints main chart (number of judgments over the judgment date)
+     * @param chart json chart data, see pl.edu.icm.saos.webapp.chart.Chart 
+     * @param xmin min x value that will be shown on the chart, null - the min x value from chart data
+     * @param xmax max x value that will be shown on the chart, null - the max x value from chart data
+     */
+    function printMainChart(chart, xmin, xmax) {
+
+        var yMinMax = calculateYAxisRangeForXRange(chart, xmin, xmax);
+        var ymin = yMinMax[0];
+        var ymax = yMinMax[1];
+        
+        var seriesArr = [];
+        
+        for (var i = 0; i < chart.seriesList.length; i++) {
+            seriesArr.push({color: colours[i], label: "", data: chart.seriesList[i].data});
+        }
+        
+        $.plot($("#mainChart"), seriesArr,    {
+                                                     lines: {
+                                                         steps:false,
+                                                         align: 'left'
+                                                    }, 
+                                                    xaxis: { 
+                                                        mode: "time", 
+                                                        timeformat: "%m/%y",
+                                                        minTickSize: [1, "month"],
+                                                        min: xmin,
+                                                        max: xmax
+                                                    }, 
+                                                    yaxis: {
+                                                        tickDecimals: 0,
+                                                        max: ymax,
+                                                        min: ymin
+                                                    }, 
+                                                    grid: {
+                                                        hoverable: true
+                                                    },
+                                                    selection: {
+                                                        mode: "x"
+                                                    }
+                                                      
+                                                     
+                                                }
+                                            );
     }
-	
-	
-	
-	
-	<!--$('input[id="daterange"]').daterangepicker();-->
-	
+    
+    /** Show a point tooltip on the main chart */
+    $("#mainChart").bind("plothover", function (event, pos, item) {
```

Comment:
It's better to use `on` than `bind`.

"As of jQuery 1.7, the .on() method is the preferred method for attaching event handlers to a document."
http://api.jquery.com/bind/